pkg/skaffold/deploy/helm/helm.go

```diff
@@ -264,18 +274,16 @@ func (h *Deployer) Render(ctx context.Context, out io.Writer, builds []build.Art
 	if err != nil {
 		return fmt.Errorf(versionErrorString, err)
 	}
+	if err = h.checkMinVersion(hv); err != nil {
```

Comment:
do we need this check again? Its on L245

Reply:
yeah, since `skaffold render` enters here we need to check it.